Get preferred container ref

p-galligan opened the pull request:
Adds functionality to get refs in `get_preferred_format`. Just grabs the digital object and top container refs from instances and returns them in `Processor`.

Slightly alters and extends tests for `get_instance_data` and `get_preferred_format`.

Nothing radical, just wanted to include this field for Aeon use. Fixes #59

Commit:
Get preferred container ref

Adds functionality to get refs in `get_preferred_format`. Just grabs the digital object and top container refs from instances and returns them in `Processor`. extends tests

bonniegee commented on process_request/helpers.py:
```diff
@@ -83,26 +83,30 @@ def get_instance_data(instance_list):
-        tuple: a tuple containing instance type, indicator, location, and
-            barcode for the instance.
+        tuple: a tuple containing instance type, indicator, location,
+            barcode, and container ref for the instance.
```
This is _very_ minor, but the "barcode" field is either the top container barcode or the Digital Object ID; does it make sense to clarify in the docstring?

helrond reviewed (approved):
Aside from Bonnie's catch on the barcodes thing, this looks good to me.

Commit:
update docstring

Updates docstring to differentiate that it's the container barcode or digital object id
